### DISCUSSION TITLE
BTV Lady

### DISCUSSION THREAD

**Discussion by dicheto** [iptv-org/community#479] [Show and tell]
### Channel ID

bTVLady.bg

### Stream URL (optional)

https://cdn3.gledam.xyz/hls/btv-lady/index.m3u8?e=1692278247&hash=wHtWhh82K8u1SqbP9m0mOw

### Notes (optional)

Is this link okay?

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://cdn3.gledam.xyz/hls/btv-lady/index.m3u8?e=1692278247&hash=wHtWhh82K8u1SqbP9m0mOw (offline)
